azure-devops/javascript/abstracts/corepack-node18.19.0.yaml

```diff
@@ -1,17 +1,13 @@
-steps:
-  - task: 'UseNode@1'
-    inputs:
-      version: '18.19.0'
-    displayName: 'Use Node.js 18.19.0'
+parameters:
+  - name: 'NODE_MODULES_PATH'
+    type: 'string'
 
-  - script: |
-      corepack enable
-      corepack prepare yarn@stable --activate
-    displayName: 'Enable Corepack and Install Yarn'
+steps:
+  - template: 'corepack-node18.19.0-no-cache.yaml'
 
   - task: 'Cache@2'
     inputs:
       key: "$(Agent.JobName)|yarn.lock"
```

Comment:
You will have to concatenate "yarn.lock" with the path you are receiving from parameters as well.